taskoutput/task_log.go

```diff
@@ -80,29 +71,27 @@ func (o TaskLogs) Get(ctx context.Context, env evergreen.Environment, taskOpts T
 	})
 }
 
-func (o TaskLogs) getLogName(taskOpts TaskOptions, logType TaskLogType) string {
+func (o TaskLogOutput) getLogName(taskOpts TaskOptions, logType TaskLogType) string {
 	prefix := fmt.Sprintf("%s/%s/%d/%s", taskOpts.ProjectID, taskOpts.TaskID, taskOpts.Execution, o.ID())
 
-	var filename string
+	var logTypePrefix string
 	switch logType {
 	case TaskLogTypeAgent:
-		filename = "agent"
+		logTypePrefix = "agent"
 	case TaskLogTypeSystem:
-		filename = "system"
+		logTypePrefix = "system"
 	case TaskLogTypeTask:
-		filename = "task"
+		logTypePrefix = "task"
 	default:
 		return prefix
 	}
 
-	return fmt.Sprintf("%s/%s", prefix, filename)
+	return fmt.Sprintf("%s/%s", prefix, logTypePrefix)
 }
 
-func (o TaskLogs) getLogServiceVersion() int {
+func (o TaskLogOutput) getLogServiceVersion() int {
 	switch {
-	case o >= 0:
-		return 0
 	default:
-		return -1
+		return 0
```

Comment:
I was going to use `-1` for cedar, but it was too difficult to implement a Cedar backed `LogService` interface. So this will just start at `0` (the v0 bucket logger implementation).